bin/node/runtime/src/lib.rs

```diff
@@ -1964,6 +1964,22 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_transaction_payment_rpc_runtime_api::TransactionPaymentCallApi<
+		Block,
+		Balance,
+		Call,
+	> for Runtime {
+		fn query_call_info(
+			call: Call,
+			len: u32,
+		) -> RuntimeDispatchInfo<Balance> {
+			TransactionPayment::query_call_info(call, len)
+		}
```

Comment:
```suggestion
	impl pallet_transaction_payment_rpc_runtime_api::TransactionPaymentCallApi<
		Block,
		Balance,
	> for Runtime {
		fn query_call_info(
			opaque_call: Vec<u8>,
			len: u32,
		) -> RuntimeDispatchInfo<Balance> {
			let call = Call::decode(&mut &*opaque_call);
			TransactionPayment::query_call_info(call, len)
		}
```

I think this is what basti wants from you.

Reply:
Did it similar, but as suggested Basti, placed it within `transaction-payment`.